Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -164,16 +164,15 @@ public struct ApolloCodegenConfiguration: Codable, Equatable {
     public let operations: OperationsFileOutput
     /// The local path structure for the test mock operation object files.
     public let testMocks: TestMockFileOutput
-    /// Configures the generation of an operation manifest JSON file for use with persisted queries
-    /// or [Automatic Persisted Queries (APQs)](https://www.apollographql.com/docs/apollo-server/performance/apq).
-    /// Defaults to `nil`.
-    public let operationManifest: OperationManifestFileOutput?
+    
+    /// This var helps maintain backwards compatibility with legacy APQ config with the new
+    /// `OperationManifestConfiguration` and will be fully removed in v2.0
+    fileprivate let operationIDsPath: String?
```

Comment:
Added this and a private `apqConfig` var because moving the operation manifest options out of their existing container structs meant the current way of decoding them to handle backward compatibility didn't work so these variables allow that backward compatibility to happen in the main decoding of the over codegen configuration file.